.github/workflows/release.yaml

```diff
@@ -110,6 +124,24 @@ jobs:
           file: config/.release/install.yaml
           asset_name: install.yaml
           tag: ${{ github.ref }}
+      
+      - name: Push yaml manifest into an OCI registry
+        env:
+          COSIGN_EXPERIMENTAL: 1
+        run: |
+            cd config/.release/ && \
+            flux push artifact oci://ghcr.io/${{ github.repository_owner }}/manifests/kyverno:$(git rev-parse --short HEAD) \
+            --path="." \
+            --source="$(git config --get remote.origin.url)" \
+            --revision="(git tag --points-at HEAD)/$(git rev-parse HEAD)"
```

Comment:
@stefanprodan is the format ok here ?
@developer-guy nit: can you add a tab on args lines so that they are indented one level more that the original command ?

Reply:
Also @developer-guy don't we need `set -e` here to catch errors ?

Reply:
Revision is good now 👍

Reply:
Ah ok, i thought you expected a `|`.